src/api/api_routes.py

```diff
@@ -0,0 +1,68 @@
+# Non-public API. Don't host publicly - SECURITY RISKS!
+# (will only be on with --api starting option)
+# Currently no API stability guarantees are provided - API may break on any new commit.
+
+from fastapi import FastAPI, Body
+from fastapi.exceptions import HTTPException
+from itertools import tee
+
+import gradio as gr
```

Comment:
Can we remove it?

Reply:
we can but the empty `_` parameter must stay